### ISSUE TITLE
Update Section 3.4 of the IPFS paper to reference Filecoin

### ISSUE THREAD

**Issue by cleichner** [ipfs/papers#35]
It mentions that there could be a virtual currency for the data market that would be discussed in a future paper. That paper is out now, right?

**Comment by jbenet** [comment]
Not quite. This refers to a future paper describing how cryptocurrency can be exchanged for data in bitswap itself. (The paper you mean is the Filecoin protocol description.)—
